pkg/dbutil/common.go

```diff
@@ -208,6 +213,52 @@ func GetRandomValues(ctx context.Context, db *sql.DB, schemaName, table, column
 	return randomValue, nil
 }
 
+// GetMinMaxValue get min and max value.
```

Comment:
GetMinMaxValue return min and max value of given column by specified `limitRange` condition